### PR TITLE
[INTERNAL] Add documentation regarding bundling

### PR DESCRIPTION
**Pull request by tommyvinhlam**

### CODE REVIEW

**Comment by tommyvinhlam** [comment]
Depends on https://github.com/SAP/ui5-builder/pull/219

**Comment by tommyvinhlam** [comment]
messed it up again

File: docs/Configuration.md [hunk 1]
```diff
@@ -94,17 +95,16 @@ Some general information:
 
 #### builder (optional)
 - `resources`: General resource configuration for this project
-    - `excludes`: List of glob patterns. Matching resources of this project will be ignored by the builder (i.e. all tasks).
```
Comment (matz3) [inline docs/Configuration.md]: I think those two spaces were put there intentionally to have a line-break

File: docs/Configuration.md [hunk 2]
```diff
@@ -302,6 +302,66 @@ shims:
...
+        name: "sap-ui-custom.js"
+        defaultFileTypes:
+          - ".js"
+        sections:
+          - mode: raw
+            filters:
+            - ui5loader-autoconfig.js
+            resolve: true
+            sort: true
+        bundleOptions:
```
Comment (matz3) [inline docs/Configuration.md]: Shouldn't this be on the same indentation-level as the `bundleDefinition`?

File: docs/Configuration.md [hunk 3]
```diff
@@ -302,6 +302,66 @@ shims:
...
+            filters:
+            - ui5loader-autoconfig.js
+            resolve: true
+        bundleOptions:
+          optimize: true
+  ```
+
+### Properties
+
+**bundleDefinitions**
```
Comment (matz3) [inline docs/Configuration.md]: bundleDefinition?
This list is part of the `bundles` property

File: docs/Configuration.md (hunk 1 above)
Comment (tommyvinhlam) [reply docs/Configuration.md]: Done. I have put them back to keep the line-break

File: docs/Configuration.md (hunk 2 above)
Comment (tommyvinhlam) [reply docs/Configuration.md]: Good eyes! :D Done!

File: docs/Configuration.md (hunk 3 above)
Comment (tommyvinhlam) [reply docs/Configuration.md]: Thanks, it should be the `bundles` property

**Pull request closed without merging**